pkg/sgpp/utils.go

```diff
@@ -30,3 +38,68 @@ func LoadEnforcingModeFromEnv() EnforcingMode {
 		return DefaultEnforcingMode
 	}
 }
+
+func AddFeatureToCNINode(ctx context.Context, k8sClient client.Client, nodeName string, featureName rc.FeatureName, featureValue string) error {
+	cniNode := &rc.CNINode{}
+	key := types.NamespacedName{
+		Name:      nodeName,
+		Namespace: "default",
+	}
+	err := k8sClient.Get(ctx, key, cniNode)
+	if err != nil {
+		return err
+	}
+
+	var names []rc.FeatureName
+	for _, feature := range cniNode.Spec.Features {
+		names = append(names, feature.Name)
+	}
+	if Include(featureName, names) {
+		return nil
+	}
+
+	newCNINode := cniNode.DeepCopy()
+	newFeature := rc.Feature{
+		Name:  featureName,
+		Value: featureValue,
+	}
+	newCNINode.Spec.Features = append(newCNINode.Spec.Features, newFeature)
+	return k8sClient.Patch(ctx, newCNINode, client.MergeFromWithOptions(cniNode, client.MergeFromWithOptimisticLock{}))
+}
+
+func RemoveFeatureFromCNINode(ctx context.Context, k8sClient client.Client, nodeName string, features []rc.FeatureName) error {
```

Comment:
nit: as above, I think we can remove this function if we do not plan to call it

Reply:
Makes sense. I was hesitating the same and decided keeping it here to save the option to support disable the feature without terminating nodes. I can remove them.